lightkurve/seismology/tests/test_butler.py

```diff
@@ -70,6 +70,14 @@ def test_estimate_numax_basics():
     nmxday = u.Quantity(true_numax*u.microhertz, 1/u.day)
     assert(np.isclose(nmxday, numax, atol=.1*nmxday))
 
+    # Assert numax estimator fails when frequqencies are not uniform
+    f, p, true_numax, _ = generate_test_spectrum()
+    f += np.random.uniform(size=len(f))
+    snr = SNRPeriodogram(f*u.microhertz, u.Quantity(p, None))
+
+    with pytest.raises(ValueError):
+        numax = snr.to_seismology().estimate_numax()
```

Comment:
A ValueError could be raised for a number of reasons, so I recommend checking that we really get the exception we want here.  This can be done as follows: 
```suggestion
    with pytest.raises(ValueError) as exc:
        numax = snr.to_seismology().estimate_numax()
    assert "uniformly spaced" in str(exc)
```
(You'll also want to do that in the second test below.)

Reply:
Thanks! I had to use `str(exc.value)`, not sure if that's a version thing or not, but it works :)